src/blocks/CaseStudyParallax/index.tsx

```diff
@@ -0,0 +1,307 @@
+'use client'
+import * as React from 'react'
+
+import { BackgroundGrid } from '@components/BackgroundGrid'
+import { BackgroundScanline } from '@components/BackgroundScanline'
+import { BlockSpacing } from '@components/BlockSpacing'
+import { Button } from '@components/Button'
+import { Gutter } from '@components/Gutter'
+import { Media } from '@components/Media'
+import { QuoteIconAlt } from '@root/icons/QuoteIconAlt'
+import { Page } from '@root/payload-types'
+import { useResize } from '@root/utilities/use-resize'
+
+import classes from './index.module.scss'
+
+type Props = Extract<Page['layout'][0], { blockType: 'caseStudyParallax' }>
+
+type StickyBlockProps = Props & {
+  currentIndex: number
+}
+
+type QuoteProps = {
+  card: any
+  isVisible?: boolean
+  className?: string
+}
+
+export const QuoteBlock: React.FC<QuoteProps> = props => {
+  const { isVisible, card, className } = props
+  return (
+    <div
+      className={[isVisible && classes.isVisible, 'cols-8 grid', className]
```

Comment:
@paulpopus This needs to comprise of only 6 columns according to the figma designs

Content: 6 columns

Gap: 2 columns

Images: 8 columns (but spans to edge)

### Current:
![Screen Shot 2024-01-25 at 12 40 26 PM](https://github.com/payloadcms/website/assets/35232443/a15edca6-2e2e-41fc-8636-a53acb727861)

### Figma:
![Screen Shot 2024-01-25 at 12 40 47 PM](https://github.com/payloadcms/website/assets/35232443/640f86c4-e8c9-4647-b291-20cdc2faa737)